storage/src/journal/fixed.rs

```diff
@@ -0,0 +1,656 @@
+//! An append-only log for storing fixed length items on disk. In addition to replay, stored items
+//! can be fetched directly by their `position` in the journal.
+//!
+//! See the [variable crate](crate::journal::variable). for a Journal that supports variable length
```

Comment:
nit: recommend putting this in italics (`_<>_`)

Reply:
done